pkg/utils/applicationprofile.go

```diff
@@ -138,3 +162,24 @@ func GetApplicationProfileContainer(object *v1beta1.ApplicationProfile, containe
 	}
 	return nil
 }
+
+func MergePolicy(policy1, policy2 v1beta1.RulePolicy) v1beta1.RulePolicy {
```

Comment:
I am sure you can do better than this...

Reply:
DONE